tests/python/gaia-ui-tests/gaiatest/apps/camera/app.py

```diff
@@ -59,6 +81,21 @@ def take_photo(self):
         # Wait for thumbnail to appear
         self.wait_for_thumbnail_visible()
 
+    def take_photo_with_timer(self):
+        # Wait for camera to be ready to take a picture
+        controls = self.marionette.find_element(*self._controls_locator)
+        Wait(self.marionette, timeout=20).until(
+            lambda m: controls.get_attribute('data-enabled') == 'true')
+         
+        self.tap_capture()
+        time.sleep(1)
```

Comment:
find_element will wait by itself until the element is found or it times, out, see:
http://mxr.mozilla.org/mozilla-central/source/testing/marionette/driver/marionette_driver/marionette.py#1574
So this sleep() call is kind of useless then anyway.